frontend/src/Components/TheMap/TheMap.tsx

```diff
@@ -97,6 +98,7 @@ export const TheMap = ({
   layerAttributeInfoIcons?: (nodeData: TreeNodeType) => ReactNode;
   allowMultipleLayerSelection?: boolean;
   showLayerManager?: boolean;
+  selectedLayerIds?: string[];
```

Comment:
Just as a reminder. 

Later we might be able to change the code a bit to only use the `selectLayerDispatchAction`. This is because it seems redundant to have both this props. But Since the initial implementation does not support multiple selection for it can be set like it is for now.

Reply:
Can you please give an example how to use `selectLayerDispatchAction` in order to set the selected layers from an upper component?